src/Orc.Extensibility/Services/SinglePluginService.cs

```diff
@@ -95,9 +96,10 @@ public async Task<IPlugin> ConfigureAndLoadPluginAsync(string expectedPlugin, st
                 }
             }
 
-            var fallbackPlugin = (from plugin in plugins
-                                  where string.Equals(plugin.FullTypeName, defaultPlugin)
-                                  select plugin).FirstOrDefault();
+            var fallbackPlugin = _defaultPlugin ?? (from plugin in plugins
```

Comment:
I think this should be the other way around. Try to load the plugin, otherwise fallback to ""default plugin" (which should actually be named "FallbackPlugin")